### ISSUE TITLE
Update to GNOME runtime 44 has broken application

### ISSUE THREAD

**Issue by neildarlow** [flathub/io.bassi.Amberol#6]
Running Amberol from the command line gives:
`[neil@vivobook ~]$ flatpak run io.bassi.Amberol
thread 'main' panicked at 'Failed to retrieve template child. Please check that it has been bound and has a #[template_child] attribute.', /run/build/amberol/vendor/gtk4/src/subclass/widget.rs:1305:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
`

**Comment by ebassi** [comment]
Indeed; this should have been fixed by #7.

Sorry about that.

**Comment by neildarlow** [comment]
Still receiving the error after revert.

```
[neil@vivobook ~]$ flatpak info io.bassi.Amberol

Amberol - Plays music, and nothing else

          ID: io.bassi.Amberol
         Ref: app/io.bassi.Amberol/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 0.9.2
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 5.5 MB
     Runtime: org.gnome.Platform/x86_64/43
         Sdk: org.gnome.Sdk/x86_64/43

      Commit: 7c3fe5174ed0e0295a421943d37dccb404c16256ad30285a71d77c7af08cc11b
      Parent: 704d44a32738bfaf03762183af27cf6706a83ee964bd2004f68ea21e986d2f41
     Subject: Roll back to GNOME 43 (689a1856)
        Date: 2023-04-24 19:25:01 +0000

`[neil@vivobook ~]$ flatpak run io.bassi.Amberol
thread 'main' panicked at 'Failed to retrieve template child. Please check that it has been bound and has a #[template_child] attribute.', /run/build/amberol/vendor/gtk4/src/subclass/widget.rs:1305:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by ghost** [comment]
Same problem here. Just finished installing Fedora 38 and Amberol.

**Comment by ebassi** [comment]
Just released 0.10.0, which should fix the issue.
